contrib/aws/aws-sdk-go-v2/aws/aws.go

```diff
@@ -39,25 +40,10 @@ func init() {
 	telemetry.LoadIntegration(componentName)
 }
 
+// duplicate tags that will be phased out in favor of aws_service & region
 const (
-	// duplicate tags that will be phased out in favor of aws_service & region
-	tagAWSService = "aws.service"
-	tagAWSRegion  = "aws.region"
-)
-const (
-	tagAWSAgent         = "aws.agent"
-	tagService          = "aws_service" //Service aws-sdk request is heading to, ex. S3, SQS
-	tagAWSOperation     = "aws.operation"
-	tagRegion           = "region" //AWS Region used to pivot from AWS Integration metrics to traces
-	tagAWSRequestID     = "aws.request_id"
-	tagQueueName        = "queuename"
-	tagTopicName        = "topicname"
-	tagTargetName       = "targetname"
-	tagTableName        = "tablename"
-	tagStreamName       = "streamname"
-	tagBucketName       = "bucketname"
-	tagRuleName         = "rulename"
-	tagStateMachineName = "statemachinename"
+	tagOldAWSService = "aws.service"
+	tagOldAWSRegion  = "aws.region"
```

Comment:
Should these go into `internal/tags`? I realize that these are old, but since the rest are moving there I think it makes sense to keep them all in one place, especially since they overlap between v1 and v2.